src/Main.hs

```diff
@@ -1,12 +1,13 @@
-import qualified Data.Text as T (strip, pack, unpack)
-
 main = putStrLn "It works!"
 
-wrap :: String -> [String]
-wrap = map trim . combine . splitAt 80
+wrap :: Int -> String -> [String]
+wrap n s = foldl (splitWordsAt n) [] . words $ s
```

Comment:
I'd probably write the last bit of this as `$ words s` instead of `. words $ s`. Or do `.words` and leave off the `s`! Point freeeee